### ISSUE TITLE
Bout time sometimes goes past 4:00.

### ISSUE THREAD

**Issue by A-Bertelsen** [A-Bertelsen/Combat-calculator#1]
Bout time sometimes goes past 4:00.

**Comment by A-Bertelsen** [comment]
This might be related to the Tick_Tock() function.

```
void Tick_Tock(){
  if(millis()>= last_millis+1000){//1000millis has passed, update timers
    UpdateTimers();
    last_millis=millis();
  }
}
```

Would be better if it was written like this
```
void Tick_Tock(){
  int now = 0;
  now = millis();
  
  if(now >= last_millis+1000){ //1000millis has passed, update timers
    UpdateTimers();
    last_millis = now;
  }
}
```

That would stop drift in the function.

The error might also be hiding somewhere else

**Comment by A-Bertelsen** [comment]
Fixed in v1.5
